ethereum/executionclient/src/integration-test/java/tech/pegasys/teku/ethereum/executionclient/web3j/Web3JExecutionEngineClientTest.java

```diff
@@ -245,19 +245,26 @@ public void newPayloadV3_shouldBuildRequestAndResponseSuccessfully() {
     // sanity check
     assertThat(executionPayloadV3Parameter.get("parentHash"))
         .isEqualTo(executionPayloadV3.parentHash.toHexString());
-    assertThat(executionPayloadV3Parameter.get("dataGasUsed"))
-        .isEqualTo(
-            Bytes.ofUnsignedLong(executionPayloadV3.dataGasUsed.longValue()).toQuantityHexString());
-    assertThat(executionPayloadV3Parameter.get("excessDataGas"))
-        .isEqualTo(
-            Bytes.ofUnsignedLong(executionPayloadV3.excessDataGas.longValue())
-                .toQuantityHexString());
-    assertThat(((List<Object>) requestData.get("params")).get(1))
-        .asInstanceOf(LIST)
-        .containsExactlyElementsOf(
-            blobVersionedHashes.stream()
-                .map(VersionedHash::toHexString)
-                .collect(Collectors.toList()));
+
+    if (isDeneb) {
+      assertThat(executionPayloadV3Parameter.get("dataGasUsed"))
+          .isEqualTo(
+              Bytes.ofUnsignedLong(executionPayloadV3.dataGasUsed.longValue())
+                  .toQuantityHexString());
+      assertThat(executionPayloadV3Parameter.get("excessDataGas"))
+          .isEqualTo(
+              Bytes.ofUnsignedLong(executionPayloadV3.excessDataGas.longValue())
+                  .toQuantityHexString());
+      assertThat(((List<Object>) requestData.get("params")).get(1))
+          .asInstanceOf(LIST)
+          .containsExactlyElementsOf(
+              blobVersionedHashes.get().stream()
+                  .map(VersionedHash::toHexString)
+                  .collect(Collectors.toList()));
+    } else {
+      // pre-deneb param must be null
```

Comment:
can change to `pre-deneb blob versioned hashes param must be null` for more clarity